ev3dev2/led.py

```diff
@@ -413,7 +415,7 @@ def _animate_police_lights():
                     self.set_color(group1, color2)
                     self.set_color(group2, color1)
 
-                if self.animate_thread_stop or stopwatch.value_ms >= duration_ms:
+                if self.animate_thread_stop or (duration is not None and stopwatch.value_ms >= duration_ms):
```

Comment:
checking for `duration_ms is not None and ...` would read a little better here since the next part of the `and` deals with `duration_ms`.

The rest looks good to me.  Thank you for the PR.

Reply:
I was thinking about it too, but I decided to compare with duration, as "duration is none --> no time limit specified, don't check elapsed time". But if you both feel like this is better, I don't have problem with it. Edited